src/components/catalog/views/targeted-offer/OfferBubbleView.tsx

```diff
@@ -10,7 +10,7 @@ export const OfferBubbleView = (props: { offer: TargetedOfferData, setOpen: Disp
     if (!offer) return;
 
     return <LayoutNotificationBubbleView fadesOut={ false } onClose={ null } onClick={ evt => setOpen(true) } gap={ 2 }>
-        <Base className="nitro-targeted-offer-icon" style={ { backgroundImage: `url(${ GetConfiguration('image.library.url') + offer._Str_13452 })` } }/>
+        <Base className="nitro-targeted-offer-icon" style={ { backgroundImage: `url(${ GetConfiguration('image.library.url') + offer.imageUrl })` } }/>
```

Comment:
This change shouldn't make it into this PR since it's related to a different PR

Reply:
Made ! 👍